docs/modules/language-tutorial/pages/02_filling_out_a_template.adoc

```diff
@@ -372,11 +372,11 @@ pipelines {
 <1> There is no pipeline object to amend. The `new` keyword gives you an object to amend.
 
 So far, you've defined objects the same way you amended them.
-When the name `foo` didn't occur before, `foo { ... }` _creates_ a property called `foo` and assigns to it the object specified on the `...`.
-If `foo` is an existing object, this notation is an _amend expression_; resulting in a new _object_ (value), but _not_ a new (named) property.
+When the `name` didn't occur before, `new { ... }` _creates_ a property called `name` and assigns to it the object specified on the `Listing`.
+If `name` is an existing object, this notation is an _amend expression_; resulting in a new _object_ (value), but _not_ a new (named) property.
 Since `pipelines` is a listing, you can _add_ elements by writing expressions in an amend expression.
 
-In this case, though, there is no object to amend. Writing `myNewPipeline { ... }` defines a _property_, but listings may only include _elements_.
+In this case, though, there is no object to amend. Writing `pipelines { ... }` defines a _property_, but listings may only include _elements_.
```

Comment:
The point being made here is about what happens _inside_ `pipelines`. At the top-level, where we said `pipelines { ... }`, there _is_ an object to amend (namely, the brand new one, just created, as per the above).

Reply:
We can keep the current verbiage if that is desired.

Reply:
```suggestion
In this case, though, there is no object to amend. Writing `myNewPipeline { ... }` defines a _property_, but listings may only include _elements_.
```